includes/discovery/sensors/voltage/netagent2.inc.php

```diff
@@ -389,8 +389,8 @@
 // BATTERY Voltage
 # Set divisor and limit ranges 1 phase UPS systems
 if ($in_phaseNum == '1') {
-    $battery_voltage_oid = '.1.3.6.1.4.1.935.1.1.1.2.2.2.0';
-    $battery_voltage1 = snmp_get($device, $battery_voltage_oid, '-Oqv');
+    $battery_voltage_oid1 = '.1.3.6.1.4.1.935.1.1.1.2.2.2.0';
+    $battery_voltage1 = snmp_get($device, $battery_voltage1_oid, '-Oqv');
```

Comment:
Hi,
This does not seems right. The 1 is not at the same position in the var name ... Did you copy paste in the github GUI ? it is easier (and less error-prone) to use git in CLI ...

Reply:
Should be ``` $battery_voltage1_oid ``` in both lines 392 and 393.

Reply:
Nah, I used CLI but modified the file without really testing it as I've modified it separately. The code below https://github.com/librenms/librenms/blob/1588b75927d2137b185ab3393e83fba8db946459/includes/discovery/sensors/voltage/netagent2.inc.php#L412 references variable names with '1' in name hence I am assuming that '1' was desired by author of this code for some reason.

Reply:
Fixed it, now both 3-phase and single phase are the same (with single-phase mirroring working 3-phase)

Reply:
All good.